### PR TITLE
TINKERPOP-2020 Added withComputer() in javascript.

### PR DESCRIPTION
**Pull request by spmallette**
https://issues.apache.org/jira/browse/TINKERPOP-2020

Was able to remove all the test ignores around this. When merging forward I hope the remaining "computer" ignores can also drop: 

https://github.com/apache/tinkerpop/blob/f7aeac4320bc673ec1c875bd781a1e5eb65ff634/gremlin-javascript/src/main/javascript/gremlin-javascript/test/cucumber/feature-steps.js#L84-L98

Builds with `mvn clean install -pl gremlin-javascript`

VOTE +1

### CODE REVIEW

File: gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/traversal-strategy.js [hunk 1]
```diff
@@ -69,7 +70,32 @@ class TraversalStrategy {
...
 }
 
+class VertexProgramStrategy extends TraversalStrategy {
+
+  constructor(graphComputer, workers, persist, result, vertices, edges, configuration) {
+    super();
+    this.configuration = new Map();
+    if (graphComputer !== undefined)
+      this.configuration.set("graphComputer", graphComputer);
+    if (graphComputer !== undefined)
```
Comment (jorgebay) [inline gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/traversal-strategy.js]: I think this condition was meant to be `workers !== undefined`, the following conditions have the same problem.

Comment (spmallette) [reply gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/traversal-strategy.js]: yes, that was a mistake. fixed that. thanks

File: gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/traversal-strategy.js [hunk 2]
```diff
@@ -69,7 +70,32 @@ class TraversalStrategy {
...
+    this.configuration = new Map();
+    if (graphComputer !== undefined)
+      this.configuration.set("graphComputer", graphComputer);
+    if (workers !== undefined)
+      this.configuration.set("workers", workers);
+    if (persist !== undefined)
+      this.configuration.set("persist", persist);
+    if (result !== undefined)
+      this.configuration.set("result", result);
+    if (graphComputer !== undefined)
```
Comment (jorgebay) [inline gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/traversal-strategy.js]: Typo :)

Comment (spmallette) [reply gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/traversal-strategy.js]: that was pretty useless of me :disappointed:  - fixed (hopefully)

**Pull request merged**